### PR TITLE
#122 issue

### PR DESCRIPTION
**Pull request by Bernardoow**
I implemented the buttons mentioned in issue #122, using dropdown like @kmmbvnr told. I think kkk.

So, it's my first pull request. Maybe I made something wrong but I guess I didn't.

For this Implementation I created two css rules.

### CODE REVIEW

**Comment by kmmbvnr** [comment]
Thank you for the pr. I on vacation right now. I'll review it as soon as i return back

**Comment by kmmbvnr** [comment]
Thank you for PR. But hiding usual save button under the dropdown means that every save action will require 2 clicks.

I'm rather prefer to keep 'Save' button and made a dropdown tick without gap right on them, with 2 rest actions.

ex:

    <button type="submit">Save</button>
    <a ...><i class="material-icons right">arrow_drop_down</i></a>
     <ul id='saveDropdown' class='dropdown-content'>
         ...

File: material/static/material/css/forms.css [hunk 1]
```diff
@@ -212,3 +212,12 @@ select.material-ignore {
         padding: 0 0.8rem  !important;
     }
 }
+
+
+i.btn.btnDropDownAdmin{
```
Comment (kmmbvnr) [inline material/static/material/css/forms.css]: camel-case is not used in materialzecss.

please use `-` as separator

**Comment by Bernardoow** [comment]
Ok.. sorry for that. Tomorow I'll fix it. And resend pr.

**Comment by Bernardoow** [comment]
@kmmbvnr 

I made some improvement. I try hover on dropdown but in small screen don't worked nicely.
I put arrow_drop_down on dropdown and css rules were renamed.
I sliced the save button from others. The others continues on dropdown.

I'm new on materializecss and pr. Excuse me for my mistakes.

**Comment by kmmbvnr** [comment]
that's obviously broken solution


change city django site admin

**Comment by kmmbvnr** [comment]
Added additional dropdown on the left

change sea django site admin

**Pull request closed without merging**